concepts/ofd/index.md

```diff
@@ -0,0 +1,372 @@
+
+A percentage of both the ATN staking rewards coming from transaction fees and NTN rewards coming from the Newton inflation mechanism are allocated for proposer incentivisation.
+
+Only signatures that contribute further than the $\frac{2}{3}$ voting power are incentivized. In other words:
+
+- if the proposer provided an empty $ActivityProof$ and we are not at the first $\Delta$ block of the epoch, the proposer is considered as faulty and does not receive any incentive.
+- if the proposer includes signatures for exactly $\frac{2}{3}$ of the voting power, the $ActivityProof$ is valid BUT the proposer does not receive any incentive.
+- if the proposer includes signatures for more than $\frac{2}{3}$ of the voting power, then he starts to get incentives.
+
+:::
+
+## OFD economics
+
+There are two aspects to omission fault detection economics of slashing: inactivity penalties for _offending validators_ and proposer rewards for _block proposers_.
+
+### Inactivity penalties
+
+The economic loss to consensus committee members and their delegators from inactivity penalties covers loss of staking and Newton inflation rewards, block proposer rewards, jailing, and stake slashing.
+
+| Economic loss | Receiving account | Distribution | Description |
+|:-- |:--|:--|:--|
+| Loss of current epoch staking rewards | [`withheldRewardsPool`](/reference/genesis/#config.autonity-object) account  | epoch end | The _offending validator_ loses a % of staking rewards proportional to its _inactivity score_ in the epoch the omission penalty is applied. The forfeited rewards are transferred to the _withheld rewards pool_ account for community funding. Amount is determined by the [Rewards withholding calculation](/concepts/ofd/#rewards-withholding-calculation). |
+| Loss of current epoch Newton inflation rewards | [`withheldRewardsPool`](/reference/genesis/#config.autonity-object) account  | epoch end | The _offending validator_ loses a % of Newton inflation rewards proportional to its _inactivity score_ in the epoch the omission penalty is applied. The forfeited rewards are transferred to the _withheld rewards pool_ account for community funding. Amount is determined by the [Rewards withholding calculation](/concepts/ofd/#rewards-withholding-calculation). |
+| Loss of current epoch block proposer rewards | n/a  | n/a | The _block proposer_ does not earn proposer rewards for a block outside the [_delta_ $\Delta$](/concepts/ofd/#delta-delta) if it includes an [_activity proof_](/concepts/ofd/#activity-proof) that does not contain signatures for $> \frac{2}{3}$ of block quorum voting power. The forfeited proposer rewards are simply not deducted from the total ATN and NTN rewards earned for the epoch and so remain in the ATN and NTN rewards distributed to the committee stake delegators at epoch end. See [Proposer reward calculation](/concepts/ofd/#proposer-reward-calculation) formula.|
```

Comment:
```suggestion
| Loss of current epoch block proposer rewards | n/a  | n/a | The _block proposer_ does not earn proposer rewards for a block if it includes an [_activity proof_](/concepts/ofd/#activity-proof) that does not contain signatures for $> \frac{2}{3}$ of block quorum voting power. See [Proposer reward calculation](/concepts/ofd/#proposer-reward-calculation) formula.|
```

implementation detail + removing imprecise sentence. If a validator doesn't put in any effort, his part of rewards will simply be distributed among the other proposers. I.e. that sum doesn't get re-injected in the epoch rewards.

Reply:
In the edge case that the total effort of the entire network is 0 (basically impossible) the entire sum of proposer rewards will be redistributed at the next epoch (always following the proposer rewards logic). However it is a so rare edge case I don't think it is even worth it to mention it in the docs. It will just confuse people .